src/components/buttons/ButtonComponent.tsx

```diff
@@ -17,15 +18,17 @@ export const ButtonComponent: FC<ButtonProps> = ({
   textColor,
   borderColor,
   bgHover,
+  leftIcon
 }) => {
   return (
     <Button
-      width={"140px"}
+      width={"70%"}
```

Comment:
Please do not edit the file outside your feature directory, since it is a shared component.
If you want to modify it you can copy the styling and create your own component

Reply:
Ok. Since I don't know at first. I really apologize krub. I will edit as soon.

Reply:
Thank uu, just edit the file and commit again, that should be alright

Reply:
I already edit them and commit again.